cobaya/likelihoods/base_classes/planck_clik.py

```diff
@@ -52,8 +52,8 @@ def initialize(self):
             raise LoggedError(
                 self.log, "No path given to the Planck likelihood. Set the "
                           "likelihood property 'path' or the common property "
-                          "'%s'.", _packages_path)
-        clik = is_installed_clik(path=self.path_clik, allow_global=allow_global)
+                          "'%s'.", "packages_path")
```

Comment:
nor grabbed from conventions?

Reply:
I've tried to remove all non-private variables with underscore names for style consistency and to reduce the very verbose import statements. Some I renamed, maybe we should do this for packages path. 
The idea of using TypedDict was that the static consistency checking that originally came from using imported _var names is now available via the typed dict.

Reply:
> I've tried to remove all non-private variables with underscore names for style consistency and to reduce the very verbose import statements.

I really do not think this is worth losing the ability to refactor much more easily by changing definitions in a single place, and it is in fact duplicated code in the form of duplicated string definitions. If verbosity and underscores are the problem, I would propose something like `import cobaya.conventions as conv`, and then use `conv.[whatever]`.

> The idea of using TypedDict was that the static consistency checking that originally came from using imported _var names is now available via the typed dict.

Not for every value, right? because not everything is a type (neither it should be).